qiskit_sphinx_theme/translations.py

```diff
@@ -1,6 +1,6 @@
 # This code is part of Qiskit.
 #
-# (C) Copyright IBM 2018, 2023.
+# (C) Copyright IBM 2021, 2023.
```

Comment:
what is this change for?

Reply:
Looking at Git history, the file was first introduced 2021, not 2018. So I fixed it to not lie.